src/webapp/components/RequestPopup.js

```diff
@@ -67,19 +64,6 @@ const RequestPopup = ({ requests, closePopup }) => {
             fontSize="small"
             className={classes.closeIcon}
           />
-          <Typography variant="h6">
-            {meta.slackPermalink ? (
-              <Link
-                href={meta.slackPermalink}
-                underline="always"
-                target="_blank"
-              >
-                {meta["First Name"] || ""}
```

Comment:
same thing here - we want the name to show up, but just not be linked to the slack permalink

Reply:
Done